src/main/java/at/ac/tuwien/damap/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -876,13 +882,14 @@ public void composeTableReusedDatasets(XWPFTable xwpfTable){
                 insertTableCells(xwpfTable, newRow, docVar);
             }
             xwpfTable.removeRow(xwpfTable.getRows().size() - 1);
+            xwpfTable.removeRow(1);
+            replaceTableVariables(xwpfTable, replacements);
         } else {
-            //clean row
             ArrayList<String> emptyContent = new ArrayList<String>(Arrays.asList("", "", "", "", "", ""));
             insertTableCells(xwpfTable, xwpfTable.getRows().get(xwpfTable.getRows().size() - 1), emptyContent);
```

Comment:
Can these be removed as the table and the paragraph will be removed anyway a line after?